src/types/form.ts

```diff
@@ -286,7 +286,7 @@ export interface IEncryptedForm extends IForm {
   payments_channel: FormPaymentsChannel
   payments_field: FormPaymentsField
   business?: FormBusinessField
-  emails?: never
+  emails: string[] | string
```

Comment:
I recall that we wanted to make it `string[]` only

Reply:
Thanks for the catch!